make_genes_and_identify_all_targets.py

```diff
@@ -180,10 +181,24 @@ def parse_additional(self, descr, input_path):
         if not any(self.origin_short_filename.startswith(r) for r in self.resistances):
             raise RuntimeError("Unconventional flash_resistance '{}' for flash_key '{}'".format(raw_resistance, self.key))
 
+    def parse_generic(self, descr):
+        self.origin = 'generic'
+        dl = descr.split("|")
+        self.key = None
+        for part in dl:
+            if part.startswith("flash_key"):
+                self.key = part.split(":", 1)[1]
```

Comment:
I'm not sure if ":" are supposed to be allowed in names or not (I don't see it in the non-allowed check in `sanatize`, but it's probably safer to do:
```
self.key = part.split("flash_key:", 1)[1]
```
so that it's just splitting on the exact key expected.